### PR TITLE
Allow CLI options to be passed to the new statinamic start/build

### LINKED ISSUE #262: [next] CLI options cannot be passed with new command
`statinamic start --devPort=4000` throw an error. commander try to validate all option, but we already use minimist, later in the process.
We probably need to ditch commander.

### PR DESCRIPTION
**Pull request by MoOx**
Closes #262

### CODE REVIEW

File: __tests__/cli.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+const statinamic = "./node_modules/.bin/statinamic"
+
+test("should throw if a CLI flag is NOT recognized", async (t) => {
+  return (
+    asyncExec(`${ statinamic } start --lol`, execOpts)
+    .then(
+      () => t.fail(),
+      (err) => {
+        t.ok(err.message.indexOf("nknown option `--lol'") > -1)
```
Comment (thangngoc89) [inline __tests__/cli.js]: *unknow ?

Comment (MoOx) [inline __tests__/cli.js]: Oops. But this won't change the test, so will update another time :)

**Comment by thangngoc89** [comment]
Oops 

```
> Statinamic@ start /home/khoa/web/statinamic/docs
> statinamic start

/home/khoa/web/statinamic/lib/builder/index.js:85
    throw new Error("You need to specify --static or --server");
    ^

Error: You need to specify --static or --server
    at exports.default (index.js:78:11)
    at Object.<anonymous> (build.js:12:1)
    at Module._compile (module.js:413:34)
    at loader (/home/khoa/web/statinamic/node_modules/babel-register/lib/node.js:126:5)
    at Object.require.extensions.(anonymous function) [as .js] (/home/khoa/web/statinamic/node_modules/babel-register/lib/node.js:136:7)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/home/khoa/web/statinamic/lib/bin/runner-cmd.js:12:1)
```

**Comment by thangngoc89** [comment]
```js
config = {
  server: false,
  static: false,
}
```

File: src/bin/runner.js [hunk 2]
```diff
@@ -5,19 +5,21 @@ const cwd = process.cwd()
...
+
     spawn(
       "node",
       [
         join(__dirname, "runner-cmd.js"),
-        join(cwd, script),
-        ...options.length === 0 ? defaultOptions : options,
+        join(cwd, argv.script),
+        // pass collected args, or default if there is none
+        ...args.length ? args : defaultOptions,
```
Comment (thangngoc89) [inline src/bin/runner.js]: Need revision here.

`args.length` always true. In my case, it is : 

```
/home/khoa/.nvm/versions/node/v5.8.0/bin/node /home/khoa/web/statinamic/docs/node_modules/.bin/statinamic
```

Comment (MoOx) [inline src/bin/runner.js]: This works for the tests. How are you using this?

Comment (MoOx) [inline src/bin/runner.js]: Nvm, it's not good enough. Will review this.

Comment (thangngoc89) [inline src/bin/runner.js]: I use this the normal way. `statinamic build` and it causes the above error.

Comment (thangngoc89) [inline src/bin/runner.js]: Possible solution is merge defaultOptions with args

**Comment by thangngoc89** [comment]
> This works for the tests. How are you using this?

Tests failed but Travis status is green 

https://travis-ci.org/MoOx/statinamic/builds/116612946#L3504-L3518

**Comment by MoOx** [comment]
And __tests__/cli.js is not even executed haha.

**Comment by MoOx** [comment]
PR updated.

**Comment by thangngoc89** [comment]
:+1: lgtm

**Pull request merged**